src/main/java/uk/gov/pay/directdebit/mandate/dao/MandateDao.java

```diff
@@ -120,16 +120,16 @@
 
     @SqlUpdate("UPDATE mandates m SET state = :state FROM gateway_accounts g WHERE m.payment_provider_id = :providerMandateId " +
             "AND g.id = m.gateway_account_id AND g.organisation = :goCardlessOrganisationId AND g.payment_provider = :provider")
-    int updateStateByPaymentProviderMandateIdAndOrganisationId(@Bind("provider") PaymentProvider paymentProvider,
-                                                               @Bind("goCardlessOrganisationId") GoCardlessOrganisationId goCardlessOrganisationId,
-                                                               @Bind("providerMandateId") PaymentProviderMandateId paymentProviderMandateId,
-                                                               @Bind("state") MandateState mandateState);
+    int updateStateByPaymentProviderMandateId(@Bind("provider") PaymentProvider paymentProvider,
```

Comment:
Could this method name include something about the organisation id to help differentiate it from the other one?

Reply:
I intended for them to have different names but I was cursed by not paying enough attention to an IDE rename that went too far! I’ve updated the name.